api/kerberos.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+import os
+
+
+def do_kinit():
+    """
+    Function performs kinit with the already mounted keytab
+    :return: None
+    """
+    if os.environ['RUN_ENV'] == 'production':
```

Comment:
i'd rather have this available wherever you might have a keytab. so this would make more sense to me as:
```suggestion
    if "KERBEROS_KEYTAB" in os.environ:
```

also, a nit, best to consistently quote strings with double quotes (unless there's a really good reason). then you'll never wonder if `''` is two single quotes or a double quote.

Reply:
Done and done!